src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public abstract class Figure implements AreaCalculator, Drawable {
+    private String color;
+    private double area;
```

Comment:
The `area` field is declared but not used in this class. If it's intended to store the area of the figure, consider calculating and setting it within the class methods or remove it if it's unnecessary.